plugin/pkg/admission/security/podsecuritypolicy/admission_test.go

```diff
@@ -693,87 +862,137 @@ func TestAdmitHostIPC(t *testing.T) {
 	}
 }
 
-func TestAdmitSELinux(t *testing.T) {
-	createPodWithSELinux := func(opts *kapi.SELinuxOptions) *kapi.Pod {
-		pod := goodPod()
-		// doesn't matter if we set it here or on the container, the
-		// admission controller uses DetermineEffectiveSC to get the defaulting
-		// behavior so it can validate what will be applied at runtime
-		pod.Spec.SecurityContext.SELinuxOptions = opts
-		return pod
-	}
+func createPodWithSecurityContexts(podSC *kapi.PodSecurityContext, containerSC *kapi.SecurityContext) *kapi.Pod {
+	pod := goodPod()
+	pod.Spec.SecurityContext = podSC
+	pod.Spec.Containers[0].SecurityContext = containerSC
+	return pod
+}
 
-	runAsAny := restrictivePSP()
+func TestAdmitSELinux(t *testing.T) {
+	runAsAny := permissivePSP()
 	runAsAny.Name = "runAsAny"
 	runAsAny.Spec.SELinux.Rule = extensions.SELinuxStrategyRunAsAny
+	runAsAny.Spec.SELinux.SELinuxOptions = nil
 
-	mustRunAs := restrictivePSP()
+	mustRunAs := permissivePSP()
```

Comment:
changed this to start with a permissive PSP and tighten just the SELinux aspect of it to isolate tests